gitspy/RepositoryCollectionViewCell.swift

```diff
@@ -10,7 +10,10 @@ import UIKit
 
 class RepositoryCollectionViewCell: UICollectionViewCell {
     
-    @IBOutlet weak var repositoryTitleLabel: UILabel!
+    @IBOutlet private weak var repositoryTitleLabel: UILabel!
     
+    func setRepository(repository: Repository?) {
+        repositoryTitleLabel.text = repository?.name
+    }
```

Comment:
ok :+1: anyway the UI is just a draft  to test the APIs ATM (right?), I can revert this part